### ISSUE TITLE
Syllabi page and courses page has IA problems since they are confusingly similar. 

### ISSUE THREAD

**Issue by kanarinka** [engagementgamelab/boston-civic-media#75]
We may end up not having time for this but it would be great to show the faculty name and their institution along with the course title. One of the things that we want to communicate with the home page is that this is a cross-institutional endeavor and also that the courses are not "ours" in the sense that BCM runs its own courses.

**Comment by jpailthorpe** [comment]
I agree. I addressed this problem with the copy and framing of the home page many weeks ago. This is a multipronged issue that has lots to do with site IA...aka syllabi and courses are confusing, since they aren't the same thing. when originally designing this, Aidan didn't change any info on the home page below the hero, because is wasn't communicated that we needed to replace the syllabi with courses. 

Again, this is just one of those things we both raised a few weeks ago and got pushback as a complete ia overview being out of scope. We made some of these changes anyway to make the site make more sense, but this one was too big to tackle. This is why we separated courses into its own top header (originally there was NO way to access the course information from the main nav at all!). 

This can't be addressed today though.

**Comment by jpailthorpe** [comment]
see #30  and #94 for additional info on this issue. 

we did change the header to now read "climate courses" to help distinguish these things, though context above will help even more when there is time.

**Comment by kanarinka** [comment]
This issue that I posted got retitled by Jordan so I am reposting it as a separate issue. The one here can remain in icebox
